app/helpers/application_helper/button/miq_action_modify.rb

```diff
@@ -16,4 +16,8 @@ def calculate_properties
                      end
     end
   end
+
+  def visible?
+    x_active_tree != :event_tree && role_allows?(:feature => 'event_edit')
```

Comment:
@vecerek You need to use `@view_context.x_active_tree` here, otherwise you get this:

```
[----] F, [2016-10-17T17:37:19.790542 #13780:2b16ee5e2b50] FATAL -- : Error caught: [NameError] undefined local variable or method `x_active_tree' for #<ApplicationHelper::Button::MiqActionModify:0x007f56b2418998>
/home/rblanco/devel/manageiq/app/helpers/application_helper/button/miq_action_modify.rb:21:in `visible?'
/home/rblanco/devel/manageiq/app/helpers/application_helper/button/basic.rb:62:in `skipped?'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:110:in `block in build_select_button'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:93:in `each'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:93:in `each_with_index'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:93:in `build_select_button'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:222:in `build_button'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:23:in `block (2 levels) in build_by_class'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:22:in `each'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:22:in `block in build_by_class'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:15:in `each'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:15:in `each_with_index'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:15:in `build_by_class'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:70:in `build_toolbar'
/home/rblanco/devel/manageiq/app/helpers/application_helper/toolbar_builder.rb:6:in `call'
/home/rblanco/devel/manageiq/app/helpers/application_helper.rb:373:in `build_toolbar'
/home/rblanco/devel/manageiq/app/controllers/miq_policy_controller.rb:495:in `replace_right_cell'
/home/rblanco/devel/manageiq/app/controllers/miq_policy_controller.rb:273:in `tree_select'
```